ci-operator/step-registry/rhoai/tests/rhoai-tests-commands.sh

```diff
@@ -5,25 +5,42 @@ set -o errexit
 set -o pipefail
 set -o verbose
 
-OCM_ENV=$API_HOST
-SET_ENVIRONMENT="1"
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+export CONSOLE_URL
+export API_URL
+export KUBECONFIG=$SHARED_DIR/kubeconfig
+
+# login for interop testings
+OCP_CRED_USR="kubeadmin"
+export OCP_CRED_USR
+OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+export OCP_CRED_PSW
+oc login -u kubeadmin -p "$(cat $SHARED_DIR/kubeadmin-password)" "${API_URL}" --insecure-skip-tls-verify=true
+
 OC_HOST=$(oc whoami --show-server)
-CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
-CLUSTER_NAME=$(cat "${SHARED_DIR}/cluster-name")
-OCM_TOKEN=$(cat /var/run/secrets/ci.openshift.io/cluster-profile/ocm-token)
-ROBOT_EXTRA_ARGS="-i $TEST_MARKER -e AutomationBug -e Resources-GPU -e Resources-2GPUS"
```

Comment:
I think we should run with passing `ROBOT_EXTRA_ARGS` from within ods-ci, though I don't see it being set to some default value in the [ods-ci/run_interop.sh](https://github.com/red-hat-data-services/ods-ci/blob/5bd2109e56b75bb390295d54de6e5d3de61ed957/ods_ci/run_interop.sh#L11):